packages/@aws-cdk/core/lib/bundling.ts

```diff
@@ -55,6 +74,20 @@ export interface BundlingOptions {
   readonly user?: string;
 }
 
+/**
+ * Local bundling
+ *
+ * @experimental
+ */
+export interface ILocalBundling {
+  /**
+   * This method is called before attempting docker bundling to allow the
+   * bundler to be executed locally. If the local bundler exists, and bundling
+   * was performed locally, return `true`. Otherwise, return `false`.
+   */
+  tryBundle(bundleDir: string): boolean;
```

Comment:
pass `options`

Reply:
Not sure here, we know `options` when defining `bundling` (we define `bundling.docker` when creating an asset) but we need `bundleDir` to correctly bundle locally.

Reply:
Oh yes you are right, so pass `options` additionally to `bundleDir`.

Is bundleDir where the output should go or the input? I am a bit confused by the terminology.

Reply:
where the output should go, will rename to `outputDir`